ROADMAP.md

```diff
@@ -20,41 +20,45 @@ See also the [ROCm Device Support Wishlist GitHub Discussion](https://github.com
 
 ### ROCm on Linux
 
-**AMD Instinct**
+#### AMD Instinct - Linux
 
 | Architecture | LLVM target | Build Passing | Sanity Tested | Release Ready |
 | ------------ | ----------- | ------------- | ------------- | ------------- |
+| **CDNA4**    | **gfx950**  | ✅            | ✅            | ✅            |
 | **CDNA3**    | **gfx942**  | ✅            | ✅            | ✅            |
-| CDNA2        | gfx90a      |               |               |               |
-| CDNA         | gfx908      |               |               |               |
-| GCN5.1       | gfx906      |               |               |               |
-| GCN5         | gfx900      |               |               |               |
```

Comment:
I removed gfx900 for now since we don't have it listed at https://github.com/ROCm/TheRock/blob/13769588e9de654dc7efaa5c3a4f4822383b8b4c/cmake/therock_amdgpu_targets.cmake#L44-L51

We could add the whole list of targets from https://llvm.org/docs/AMDGPUUsage.html:
* gfx900
* gfx902
* gfx904
* gfx906
* gfx908
* gfx909
* gfx90a
* gfx90c